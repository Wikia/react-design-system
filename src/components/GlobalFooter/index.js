import React from 'react';

/**
 * TODO Not yet implemented
 */
const GlobalFooter = () => <div>NOT IMPLEMENTED</div>;

export default GlobalFooter;
