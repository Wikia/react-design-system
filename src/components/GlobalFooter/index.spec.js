import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import sinon from 'sinon';

import GlobalFooter from './index';

test('GlobalFooter should render correctly with default data', () => {
  const component = renderer.create(
    <GlobalFooter />
  );
  expect(component.toJSON()).toMatchSnapshot();
});
