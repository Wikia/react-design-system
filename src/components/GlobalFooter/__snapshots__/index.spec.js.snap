// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalFooter should render correctly with default data 1`] = `
<div>
  NOT IMPLEMENTED
</div>
`;
